libobs/obs.h

```diff
@@ -1575,8 +1581,29 @@ EXPORT void obs_scene_atomic_update(obs_scene_t *scene,
 EXPORT void obs_sceneitem_addref(obs_sceneitem_t *item);
 EXPORT void obs_sceneitem_release(obs_sceneitem_t *item);
 
-/** Removes a scene item. */
+/** Removes and adds a scene item. */
 EXPORT void obs_sceneitem_remove(obs_sceneitem_t *item);
+EXPORT void obs_sceneitems_add(obs_scene_t *scene, obs_data_array_t *data);
+
+/** Saves Sceneitem into arr **/
+EXPORT void obs_sceneitem_save(obs_sceneitem_t *item, obs_data_array_t *arr);
+
+/** Set the id of a sceneitem */
+EXPORT void obs_sceneitem_set_id(obs_sceneitem_t *sceneitem, int64_t id);
+
+/** Tries to find the sceneitem of the source in a given scene */
```

Comment:
Does it try or always succeed?  If it fails, what does it return?  If there's a failure case, make sure the documentation for this mentions it.